### ISSUE TITLE
v2.5.3 not working with chalk v1.1.2

### ISSUE THREAD

**Issue by huozicheng** [eslint/eslint#5716]
**What version of ESLint are you using?**
v2.5.3
**What parser (default, Babel-ESLint, etc.) are you using?**
default
**Please show your full configuration:**
{
  "extends": "name",
  "rules": {
    "no-useless-call": 1,
    "comma-dangle": 2
  }
}

**What did you do? Please include the actual source code causing the issue.**
npm install
**What did you expect to happen?**
working
**What actually happened? Please include the actual, raw output from ESLint.**
.../node_modules/gulp-eslint/node_modules/eslint/lib/formatters/stylish.js:71
                    return chalk.stripColor(str).length;
                                 ^

TypeError: chalk.stripColor is not a function
    at table.stringLength (.../node_modules/gulp-eslint/node_modules/eslint/lib/formatters/stylish.js:71:34)
    at .../node_modules/text-table/index.js:33:21
    at Array.forEach (native)
    at forEach (.../node_modules/text-table/index.js:73:31)
    at .../node_modules/text-table/index.js:32:9
    at Array.reduce (native)
    at reduce (.../node_modules/text-table/index.js:63:30)
    at module.exports (.../node_modules/text-table/index.js:31:17)
    at .../node_modules/gulp-eslint/node_modules/eslint/lib/formatters/stylish.js:46:19
    at Array.forEach (native)

Thank you

**Comment by jsdelfino** [comment]
Running into the same problem on Node 4.2 (5.9 works). Looks like a problem with the stylish formatter, as switching to the unix of compact formatter as a workaround works.

**Comment by ilyavolodin** [comment]
I just did a clean install of eslint v2.5.3 and it comes with chalk v1.1.3. Everything is working correctly for me. Which version of NPM are you using? If you are using NPM 3, it might be that chalk is required by some other dependency in your project and NPM is screwing up deduping. Can you also try doing a clean reinstall (delete node_modules directory completely).

**Comment by huozicheng** [comment]
It is chalk v1.1.2, it updated to 1.1.3? 1.1.2 is like 10 minutes ago.

**Comment by huozicheng** [comment]
https://github.com/chalk/chalk/issues/106

**Comment by huozicheng** [comment]
I think it's fixed.

**Comment by Qix-** [comment]
Yeah. We had an issue earlier today so I've been doing damage control. As long as you're not requiring *exactly* v1.1.2 it should be fixed by simply reinstalling packages.

Apologies.

**Comment by morrissinger** [comment]
I ran into this issue. It looks like the problem is that package.json isn't hard-versioned, and chalk issued a breaking change in the point-point release by removing their export of the `stripColor()` method from the strip-ansi package.

See: https://github.com/eslint/eslint/blob/master/package.json#L39

I fixed this by bringing in the strip-ansi dependency directly. See PR: https://github.com/eslint/eslint/pull/5731
